chaos-monkey-spring-boot/pom.xml

```diff
@@ -62,11 +62,27 @@
             <artifactId>lombok</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>${junit.jupiter.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Quite picky I know :see_no_evil: , but can you move that below of junit dependency (then its grouped with the other jupiter stuff)

Reply:
Sure, that's a good idea! :+1: